repository/songs_list.json

```diff
@@ -0,0 +1,8 @@
+[

```

Comment:
This file seems to serve as fake data for Infrastructure functions. Please move it in the same directory where that function is located.